src/Shared/Shared.csproj

```diff
@@ -42,4 +42,8 @@
     <InternalsVisibleToTest Include="$(AssemblyName).Tests" />
     <InternalsVisibleToDynamicProxyGenAssembly2 Include="*" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="ServerSentEvents\" />
+  </ItemGroup>
```

Comment:
Is this redundant now the files exist? It looks like it would be.

Reply:
Not sure how this slipped by. Removed.